src/scripts/main.js

```diff
@@ -358,3 +358,27 @@ const people = [
 console.log(people); // you can remove it
 
 // write your code here
+// step 1 . 1
+for (const person of people) {
+  // find table.tbody
+  const table = document.querySelector('.dashboard').firstElementChild;
```

Comment:
The code assumes that the first child of the `.dashboard` element is the table body (`tbody`). This assumption might not always be correct. Consider using `querySelector` to directly select the `tbody` element within the `.dashboard`.